salt/cloud/clouds/gce.py

```diff
@@ -78,6 +80,9 @@
         libcloud.security.CA_CERTS_PATH.append('/etc/ssl/certs/YaST-CA.pem')
     HAS_LIBCLOUD = True
 except ImportError:
+    LIBCLOUD_IMPORT_ERRORS.append("Failure importing libcloud!")
+    for line in traceback.format_exc().split("\n"):
+        LIBCLOUD_IMPORT_ERRORS.append(line)
```

Comment:
Ok, finally looking at this, `log.exception` won't work. You can only call `log.exception` from within an exception handler: [<pre>This function should only be called from an exception handler.</pre>](https://docs.python.org/3/library/logging.html#logging.exception).

Unfortunately, you can't just pass an exception to it.